.github/workflows/tests.yml

```diff
@@ -69,6 +69,6 @@ jobs:
 
       - name: Test
         env:
-          BROWSERS: preset:local
```

Comment:
Is this removed because `preset:local` is the default?

Reply:
yes :)